src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: {
+    books: [],
+  },
+  reducers: {
+    addBook: (state, action) => {
+      state.books = [action.payload, ...state.books];
+    },
+    removeBook: (state, action) => {
+      const indexOfId = state.books.indexOf(action.payload);
+      state.books.splice(indexOfId, 1);
+    },
+  },
+});
+
+export default booksSlice;
```

Comment:
I suggest you Kindly export `addBook` and `removBook` as actions👍 as these actions will become handy when updating the book state😉.
something like this
```suggestion
export default booksSlice;
export const { addBook, removeBook } = booksSlice.actions
```
``

Reply:
@Ibizugbe, the required changes are done.